index.html

```diff
@@ -2612,6 +2617,112 @@ <h3 id="applying">
+          to catalogs or stores of web applications under which categories 
+          (or category) to list this web application.
+        </p>
+        <p>
+          The <code>categories</code> member is only meant as a hint to catalogs 
+          or stores listing web applications and it is expected that these 
+          will make a best effort to find appropriate categories (or category)
+          under which to list the web application.
+        </p>
+        <p>
+          The <dfn>steps for processing the <code>categories</code>
+          member</dfn> are given by the following algorithm. The algorithm
+          takes a <var>manifest</var> as an argument. This algorithm returns an
+          array of strings or <code>undefined</code>. The array of strings MAY 
+          include only one value.
+        </p>
+        <ol>
+          <li>Let <var>categories</var> be an empty list.</li>
+          <li>Let <var>unprocessed categories</var> be the result of calling
+          the <a>[[\GetOwnProperty]]</a> internal method of <var>manifest</var>
+          with argument "<code>categories</code>".</li>
+          <li>If <var>unprocessed categories</var>is an array, then:</li>
+            <ol>
+              <li>For each <var>potential category</var> in the array:</li>
+                <ol>
+                  <li>
+                    <a>Trim</a>(<var>value</var>) and convert 
+                    <a>to ASCII lowercase</a>
+                  </li>
+                  <li>
+                    Append <var>potential category</var> to 
+                    <var>categories</var>
+                  </li>
+                </ol>
+            </ol>
+          <li>
+            Otherwise, if <var>unprocessed categories</var> is not 
+            <code>undefined</code>
+          </li>
+            <ol>
+              <li>
+                <a>Issue a developer warning</a> that the type is not 
+                supported.
+              </li>
+            </ol>
+          <li>Return <var>categories</var></li>
+        </ol>
+        <p>
+          The categories string array is case insensitive and converted 
+          to lower-case by following the processing algorithm. Thus, 
+          <code>sports</code>, <code>Sports</code>, <code>SPORTS</code>, 
+          and <code>SpOrTs</code> are all equivalent. Manifest authors 
+          SHOULD use lower-case.
+        </p>
+        <p>
+          The categories string array is a hint to catalogs and stores 
+          of where the Manifest author suggests their web app should be 
+          listed. This is hint or suggestion from the Manifest author. 
+          Like search engine keywords, catalogs and stores are not 
+          required to honor this hint.</p>
+        <p>
+          If a categories string array has multiple values, a consuming 
+          system MAY use one or more of these values.  
+        </p>
+        <p>
+          Manifest authors SHOULD use the below common categories:
```

Comment:
We probably need to make this into a registry. The use of plural and singular form seems a bit arbitrary. 

@kenchris, can you check?